lib/bookingsync/engine/model.rb

```diff
@@ -50,12 +50,8 @@ def update_token(token)
   end
 
   def update_token!(token)
-    Thread.new do
-      ActiveRecord::Base.connection_pool.with_connection do
```

Comment:
Should be safe to remove this now that devise is patched :+1:

Reply:
Thanks @Fredar! 😌

Reply:
Wowow thanks @Fredar for helping out :D